pydatastructs/miscellaneous_data_structures/tests/test_segment_trees.py

```diff
@@ -0,0 +1,82 @@
+import random
+from pydatastructs.miscellaneous_data_structures.segment_tree import SegmentTree
+
+
+def gen_range(list_size):
+    """
+    Generates range [L, R] for querying.
+    """
+    left = random.randint(0, list_size - 1)
+    right = random.randint(left, list_size - 1)
+
+    return left, right
+
+
+def random_test():
+    """
+    Random test for the Segment Tree using random list
+    of values, random range queries.
+    """
+    random_list = [random.randint(-10 ** 6, 10 ** 6) for _ in range(10 ** 5)]
+    t = SegmentTree(random_list)
+
+    for _ in range(random.randint(10 ** 3, 10 ** 6)):
```

Comment:
We don't need to test this for such large values. It consumes much time on CI unnecessarily (python being slower).

Reply:
Yes, that's the reason I have commented it out..haven't actually called the function..